src/lammpsio/data.py

```diff
@@ -394,6 +504,106 @@ def read(self):
                                 "Expected number of columns not read for mass"
                             )
                         masses[int(row[0])] = float(row[1])
+                elif "Bonds" in line:
+                    if N_bonds is not None:
+                        snap.bonds = Bonds(N_bonds, num_types=num_bond_types)
+                    _readline(f, True)  # blank line
+                    for i in range(snap.bonds.N):
+                        row = _readline(f, True).split()
+                        if len(row) < 4:
+                            raise IOError(
+                                "Expected number of columns not read for bonds"
+                            )
+                        row = [int(x) for x in row]
+                        snap.bonds.id[i] = row[0]
+                        snap.bonds.typeid[i] = row[1]
+                        snap.bonds.members[i] = row[2:4]
+
+                    # sanity check
+                    if snap.has_bonds() and numpy.any(
```

Comment:
I think this first part is not necessary? I think we can only read a Bonds section if it has that data, and so this will always be true.
```suggestion
                    if numpy.any(
```